source/resources.html.md.erb

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This file has also changed (from `/related-resources.html`) so can we add a redirect for that as well?

Reply:
Ah good point! Fixed in baee611